### PR TITLE
Category mass update action added #6690

### PR DESCRIPTION
**Pull request by papnoisanjeev**
## Issue Reference
#6690

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@papnoisanjeev 

This is **working fine**, please find the screenshot [Here](https://prnt.sc/cJpcvT-fkS3G)

**Review by Sanjaybhattwebkul — APPROVED** [review]

File: packages/Webkul/Admin/src/Routes/catalog-routes.php [hunk 1]
```diff
@@ -104,6 +104,10 @@
             'redirect' => 'admin.catalog.categories.index',
         ])->name('admin.catalog.categories.massdelete');
 
+        Route::post('categories/massupdate', [CategoryController::class, 'massUpdate'])->defaults('_config', [
```
Comment (jitendra-webkul) [inline packages/Webkul/Admin/src/Routes/catalog-routes.php:107]: Change massupdate to mass-update

File: packages/Webkul/Category/src/Http/Controllers/CategoryController.php [hunk 2]
```diff
@@ -205,6 +205,41 @@ public function massDestroy()
...
 
+     /**
+     * Mass update Category.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function massUpdate()
+    {
+        $data = request()->all();
+
```
Comment (jitendra-webkul) [inline packages/Webkul/Category/src/Http/Controllers/CategoryController.php:216]: We can use single if for both conditions

**Comment by Sanjaybhattwebkul** [comment]
@papnoisanjeev 
Please add a **mass update** option inside the **categories** when the admin **creates the role**, please take a reference from [here](https://prnt.sc/t9YSY_wB2z1g)

**Review by Sanjaybhattwebkul — APPROVED** [review]

**Comment by Sanjaybhattwebkul** [comment]
@papnoisanjeev 
This is **working fine**, please find the screenshot [Here](https://prnt.sc/FA41rUZ-ne9N)

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**